Changed setup.py scripts path from 'geograpy/bin/geograpy-nltk' to 'g…

premanandchandrasekar opened the pull request:
…eograpy2/bin/geograpy-nltk' , added data/bin directories and .gitignore file

brunobg commented:
hi @premanandchandrasekar, thanks for the PR! Can you tell me a little bit more about it? I get the path fixes, but not the following:
- why you commited build/lib.linux-x86_64-2.7/
- why geograpy2/bin/geograpy-nltk and there are several downloads there. 
- there seems to be a \0 character in a few of the files
- why your replaced ne.label() with ne.node

premanandchandrasekar commented:
hi @brunobg , thanks for your comments. I added reply comments in below. 
- why you commited build/lib.linux-x86_64-2.7/
    -- unfortunately added build/lib.linux-x86_64-2.7/ - removed. 
- why geograpy2/bin/geograpy-nltk and there are several downloads there.
   -- if i need to install it separately,  it throws No path found error for geograpy/bin/geograpy-nltk. So i added geograpy-nltk and changed path in setup.py ,  likewise in ushahidi/geograpy. Otherwise need to remove scripts from setup.py
- there seems to be a \0 character in a few of the files
   -- all \0 characters removed .
- why your replaced ne.label() with ne.node
  -- after installation, the following error raised.
  
     Installed /usr/local/lib/python2.7/dist-packages/geograpy2-0.1.0-py2.7.egg
     error :  nltk 3.1 is installed but nltk==2.0.5 is required by set(['newspaper'])
  
   -- So I installed nltk 2.0.5, but 2.0.5 doesn't have label() method. So I replaced with "node". otherwise it will throw the following error.
  
  ```
  Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/geograpy2-0.1.0-py2.7.egg/geograpy2/__init__.py", line 6, in get_place_context
  e.find_entities()
  File "/usr/local/lib/python2.7/dist-packages/geograpy2-0.1.0-py2.7.egg/geograpy2/extraction.py", line 63, in find_entities
  if ne.label() in ['GPE', 'PERSON', 'ORGANIZATION']:
  AttributeError: 'Tree' object has no attribute 'label'
  ```
